neurolib/models/multimodel/builder/base/neural_mass.py

```diff
@@ -170,9 +175,13 @@ def update_params(self, params_dict):
         :type params_dict: dict
         """
         assert isinstance(params_dict, dict)
-        self.params.update(params_dict)
+        self.params.update({k: v for k, v in params_dict.items() if "noise" not in k})
```

Comment:
Could you add a comment here, why this is necessary? User would have to avoid any parameter with label "noise" in it?

Reply:
yes, because `noise_X` (X is integer) is the key for the noise process that is expected in the mass... I mean, I guess wouldn't have a parameter called `noise`. The only parameter call `noise` would be something like noise variance, but in MultiModel this is done by setting a noise process and connecting it to the population (see `noise_input` attribute of `NeuralMass`)..
so in this function, you first update the parameter dictionary of the mass itself, and then, couple of lines below, you actually set up (update) noise parameters